tests/generic/host/returnZero/CMakeLists.txt

```diff
@@ -0,0 +1,34 @@
+# Copyright 2024 ETH Zurich and University of Bologna.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Moritz Scherer <scheremo@iis.ee.ethz.ch>
+# Philip Wiese <wiesep@iis.ee.ethz.ch>
+
+set(TEST_NAME test_host_returnZero)
+
+######## HOST Code #############################################################
+file(GLOB_RECURSE TEST_HOST_SRCS
+  "src_host/*.c"
+)
+
+add_library(${TEST_NAME}_host OBJECT ${TEST_HOST_SRCS})
+# target_include_directories(${TEST_NAME}_host PUBLIC include)
+
+# WIESEP: Set the correct ISA and ABI for the host
+target_compile_options(${TEST_NAME}_host
+  PRIVATE
+  -O2
+  -march=${ISA_HOST}
+  -mabi=${ABI_HOST}
+)
```

Comment:
While this isn't a bad solution, this feels a bit impractical - you're asking the end user to know about `ISA_HOST` and `ABI_HOST`, and any other compile flags necessary. I think splitting `runtime`/`chimera-sdk` into `devicecode` and `hostcode` would eliminate the need for this at the cost of having to link both into the executable. What do you think?

Reply:
I only ask them to know about the ISA and ABI flags, and I think it makes sense to be able to change the optimization level for the user code separately. Yes, this is also a very nice suggestion, happy to discuss in person about the technical details.

Reply:
After the discussion we agree to expose two libraries (host and device) with public flags and then link the host application code against the host library and device application code against device library.